tools/cbmc/proofs/HTTP/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 list(PREPEND cbmc_compile_includes
     ${CMAKE_SOURCE_DIR}/libraries/c_sdk/standard/https/include
     ${CMAKE_SOURCE_DIR}/libraries/c_sdk/standard/https/src/private
-    ${CMAKE_SOURCE_DIR}/libraries/3rdparty/http_parser
+    ${CMAKE_SOURCE_DIR}/libraries/coreHTTP/source/3rdparty/http_parser
```

Comment:
Is it `coreHTTP/source/dependency/3rdparty`? Or are you not at 1.0.0 in the submodule pointer?

Reply:
Still not at v1.0.0 tag. Upgrading to that can be done in a separate PR.